packages/components/pie-textarea/test/visual/pie-textarea.spec.ts

```diff
@@ -249,7 +249,6 @@ test.describe('Label and Character count:', () => {
         await mount(PieTextarea, {
             props: {
                 label: 'Label',
-                maxLength: 250,
```

Comment:
I think the names of these visual tests will need updating

Reply:
done